packages/browser-wallet/src/shared/storage/types.ts

```diff
@@ -286,12 +287,17 @@ interface CredentialSchemaProperty {
     title: string;
     type: 'string' | 'number' | string;
     description: string;
+    format?: string;
 }
 
+type CredentialSchemaAttributes = {
+    properties: Record<string, CredentialSchemaProperty>;
+} & CredentialSchemaProperty;
+
 interface CredentialSchemaSubject {
     type: string;
+    properties: { id: CredentialSchemaProperty } & { attributes: CredentialSchemaAttributes };
```

Comment:
```suggestion
    properties: { 
        id: CredentialSchemaProperty;
        attributes: CredentialSchemaAttributes;
    };
```
(Formatting not included)